spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET #index' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get users_path
+      expect(response.body).to include('Display all the users')
+    end
+  end
+
+  describe 'GET #show' do
+    let(:user) { User.create(name: 'John') }
+
+    it 'returns a successful response' do
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get user_path(user)
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes correct placeholder text in the response body' do
+      get user_path(user)
+      expect(response.body).to include('Display the specific information of the user')
+    end
+  end
+end
```

Comment:
- [ ] Kindly put the same test cases for Post index and show page to test your controller properly according to the requirements 👍